config/zu.py

```diff
@@ -0,0 +1,56 @@
+"Zulu config with language-specific information."
+
+from pynini import *
+
+GRAPHEMES = union("'", "-",
+                  "A", "B", "C", "D", "E", "F", "G", "H", "I",
+                  "J", "K", "L", "M", "N", "O", "P", "Q", "R",
+                  "S", "T", "U", "V", "W", "X", "Y", "Z",
+                  "a", "b", "c", "d", "e", "f", "g", "h", "i",
+                  "j", "k", "l", "m", "n", "o", "p", "q", "r",
+                  "s", "t", "u", "v", "w", "x", "y", "z")
+
+INITIAL_PUNCTUATION = union('"', "'")
+
+FINAL_PUNCTUATION = union("!", '"', ",", ".", ":", ";", "?")
+
+NUMBERS = union("0", "1", "2", "3", "4", "5", "6", "7", "8", "9")
+
+HYPHEN = acceptor("-")
+
+SPACE = acceptor(" ")
```

Comment:
Since we're just using this once, we don't really need to define it separately I think.  Also, you could move the HYPHEN definition to the top, so that you can call it in GRAPHEMES.

Reply:
Done!